modules/ComscoreStreamingTag/tests/test_ads.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+	<title>ComscoreStreamingTag Module - ads </title>
+	<script type="text/javascript" src="../../../tests/qunit/qunit-bootstrap.js"></script>
+	<script type="text/javascript" src="../../../mwEmbedLoader.php"></script>
+	<script type="text/javascript" src="../../../docs/js/doc-bootstrap.js"></script>
+</head>
+<body>
+<h1>ComscoreStreamingTag Plugin - advertising</h1>
+<br>
+<div id="myVideoTarget" style="float:left;padding-left:10px;width:400px;height:330px;"></div>
+	<i>Analytics events:</i> ( only html5 )<br />
+	<textarea id="analyticsLog" rows='15' style="width:340px;"></textarea>
+</div>
+
+<div style="clear:both"></div>
+
+<script>
+	var localPath = document.location.href;
+	localPath = localPath.split('VPAID.html')[0];
+	kWidget.featureConfig({
+		'targetId': 'myVideoTarget',
+		'wid': '_243342',
+		'uiconf_id' : '13306622',
+		'entry_id' : '0_uka1msg4',
+		'flashvars': {
+			'vast': {
+				'plugin' : true,
+				'debug': 'true',
+				'position': 'before',
+				'relativeTo': 'PlayerHolder',
+				'numPreroll':"1",
+				'prerollInterval':"1",
+				'prerollStartWith':"1",
+				'prerollUrl':"http://localhost:8888/modules/AdSupport/tests/liverail.xml?random={utility.random}&timestamp={utility.timestamp}&referrer_url={utility.referrer_url}&referrer_host={utility.referrer_host}",
```

Comment:
please point to public test file such as http://projects.kaltura.com/mdale/hotelVastAd.xml or use relative paths to point to one hosted in the library.